packages/grafana-ui/src/components/TimePicker/time.ts

```diff
@@ -0,0 +1,44 @@
+import moment, { Moment } from 'moment';
+import { TimeOption, TimeRange, TIME_FORMAT } from '@grafana/ui';
+
+import * as dateMath from '../../../../../public/app/core/utils/datemath';
+import { describeTimeRange } from '../../../../../public/app/core/utils/rangeutil';
```

Comment:
How is this able to use `app/core/utils` from @grafana/ui?

Should we add something to fail the build if grafana/ui imports from grafana core?

Reply:
Yup, failing the build is a good idea.